common/changes/office-ui-fabric-react/caperez-may2019-extensions_2019-05-31-21-12.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "adding missing filetype mappings to support LPC and other partners",
+      "type": "minor"
```

Comment:
@leddie24 this change file needs to be either removed or if it's impossible then change manually the type to none so it's not bumping a minor on OUFR.